File: surelock/src/main/java/com/smashingboxes/surelock/SurelockException.java
package com.smashingboxes.surelock;

/**
 * Created by Tyler McCraw on 3/5/17.
 * <p>
 *     Exception for any issues in setting up Surelock
 *     dependencies for encryption/decryption using FingerprintManager
 * </p>
 */

public class SurelockException extends RuntimeException {

    public SurelockException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: surelock/src/main/java/com/smashingboxes/surelock/SurelockException.kt
package com.smashingboxes.surelock

/**
 * Created by Tyler McCraw on 3/5/17.
 *
 *
 * Exception for any issues in setting up Surelock
 * dependencies for encryption/decryption using FingerprintManager
 *
 */
open class SurelockException(message: String, cause: Throwable?) : RuntimeException(message, cause)
